common/src/main/java/gov/cms/ab2d/common/model/Coverage.java

```diff
@@ -5,32 +5,28 @@
 import lombok.Getter;
 import lombok.Setter;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
+import javax.persistence.*;
 
 @Entity
 @Getter
 @Setter
-@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
-public class Coverage extends TimestampBase {
```

Comment:
Why was the extension of TimestampBase dropped?

Reply:
This is tied to a CoverageSearchEvent with a foreign key relationship. That event has a specific time on it. Since contract searches return all at once I think that time is fairly accurate. We will also have tens maybe hundreds of millions of these so adding those two columns adds a lot of data.

Reply:
@nflearl if you disagree I can add it back in but it just seems like more information than we necessarily need.